src/course-home/outline-tab/messages.js

```diff
@@ -99,6 +99,11 @@ const messages = defineMessages({
     defaultMessage: 'Open',
     description: 'A button to open the given section of the course outline',
   },
+  optionalCompletion: {
+    id: 'learning.outline.optionalBlock',
+    defaultMessage: 'Optional',
+    description: 'Used as a label to indicate that a section, sequence, or unit is optional.',
```

Comment:
This text is used only for sections and sequences. Units are not displayed on the Course Outline page.

Reply:
nice catch